ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/PaywallAction.kt

```diff
@@ -0,0 +1,9 @@
+package com.revenuecat.purchases.ui.revenuecatui.components
+
+import com.revenuecat.purchases.paywalls.components.ButtonComponent
+
+internal sealed interface PaywallAction {
```

Comment:
I've been thinking whether to reuse the `ButtonComponent.Action` for now, since right now we will require to map from that to this type....The idea is to have all paywall customer actions here eventually, so we can pass the action back to the view model where the logic would live to update the state. Lmk what you think!

Reply:
I think we'll have to build this eventually, as this is going to contain other actions too if I'm not mistaken? Things like "purchase", and "select package". So we might as well have it now already.